tests/performance/test_performance.cpp

```diff
@@ -393,16 +410,36 @@ class Test {
     }
 };
 
+bool
+valid_and_extrcat_top_k(const std::string& input, int64_t& number) {
+    std::regex pattern(R"(^search:(\d+)$)");
+    std::smatch match;
+    if (std::regex_match(input, match, pattern)) {
+        number = std::stoi(match[1].str());
+        if (number <= 0 || number > 100) {
```

Comment:
Is 100 is too small?

Reply:
Generally, when evaluating index performance, more attention is paid to the case where k is 100; however, this check condition can be removed.